R/edit_file.R

```diff
@@ -142,7 +142,7 @@ edit_file <- function(metadata_dir = here::here("data", "metadata"),
 
 
     ## bounding box map
-    if (file == "biblio.csv"){
+    if (file == "biblio.csv" & requireNamespace("maps", quietly = TRUE)){
```

Comment:
Two things here:

- Just as a convention, I like to use non-element-wise `&&` in places like this and reserve `&` for when I need element-wise anding. What do you think about changing this to `&&`?
- Do you think people would want a `message` to get spit out if `maps` isn't installed?

Reply:
Sure thing.  Done.

Re: a message, perhaps we can discuss this a bit further. For example it would be nice if there are bounding box details in biblio.csv that they are plotted on the map. Then it would make the map in edit_biblio a bit more useful. At the minute it just shows a map of the world if maps is installed and nothing if not. Seems not a lot of reason to suggest they install maps at the moment so for now I think it's ok.

Reply:
Oh okay. I hadn't poked around that part recently enough to remember. Sounds good.